mozilla_django_oidc_db/backends.py

```diff
@@ -37,6 +40,11 @@ def __init__(self, *args, **kwargs):
         # to avoid a large number of `OpenIDConnectConfig.get_solo` calls when
         # `OIDCAuthenticationBackend.__init__` is called for permission checks
 
+    def retrieve_identifier_claim(self, claims: dict) -> str:
+        identifier_claim_name = getattr(self.config, self.config_identifier_field)
+        unique_id = glom(claims, identifier_claim_name, default="")
```

Comment:
note: this will cause problems if the claim name itself contains a dot, e.g. 

```json
{
    "foo.bar": "baz"
}
```
probably we need to just cover this in documentation/help text and fix it when we run into it

Reply:
I don't rule it out, I don't expect it to be an immediate problem either. So I'm in favour of "let's deal with it when it does become an issue", but now we're at least prepared and it won't come as a complete surprise if we get a bug report because of it :)

Reply:
Documented in the code and readme in https://github.com/maykinmedia/mozilla-django-oidc-db/pull/43/commits/dfe17a8d942e7755f5aca953078888bbaa23dde0